invenio_mail/ext.py

```diff
@@ -76,3 +78,6 @@ def init_config(app):
         """
         app.config.setdefault("MAIL_DEBUG", app.debug)
         app.config.setdefault("MAIL_SUPPRESS_SEND", app.debug or app.testing)
+        for k in dir(config):
+            if k.startswith("MAIL_"):
```

Comment:
this will shadow flask-mail config variables and potentially set some defaults that we won't like :), either we need a default only for this particular variable, or to choose another prefix for naming for this module. I would personally go with specifc prefix.
We could discuss over the teams chat, which name would work best

Reply:
Discussed in real life. Settled on being ok with this change, since the last update of flask mail dates back a few years and config variables are not expected to change any time soon.